packages/@sanity/base/src/change-indicators/overlay/Connector.styled.tsx

```diff
@@ -0,0 +1,87 @@
+import {Theme} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+import {ClampedRect} from './ClampedRect'
+
+interface PathInterface {
+  focused: boolean
+  revertedHovered: boolean
+  hovered: boolean
+  theme: Theme
+}
+
+export const DebugRectWrapper = styled.rect`
+  stroke: #ccc;
+  fill: none;
+  pointer-events: none;
+  stroke-linecap: round;
+`
+
+export const PathWrapper = styled.path(
```

Comment:
Same as above, since it's not strictly _wrapping_ a path element, but rather styling it, maybe it could be named "ConnectorPath", or "StyledPath" or something?